### ISSUE TITLE
prepData() function is abscent

### ISSUE THREAD

**Issue by kalmSveta** [HelenaLC/CATALYST#136]
I have just installed CATALYST via Bioconducter. I was trying to run it on a test dataset:
```
library(CATALYST)
data("raw_data")
sce <- prepData(raw_data)
```

However got an error:
`Error in prepData(raw_data) : could not find function "prepData"`

`sessionInfo()`
```
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.5 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.7.1
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.7.1

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=ru_RU.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=ru_RU.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=ru_RU.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=ru_RU.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] CATALYST_1.8.7
```

**Comment by HelenaLC** [comment]
The current release version of `CATALYST` is actually is 1.12 with Bioconductor 3.11, which requires R > 4. I recommend you update your R version, and install the stable release via `install.packages("BiocManager"); BiocManager::install("CATALYST")`

**Comment by kalmSveta** [comment]
Than you very much, I will try that

________________________________
От: Helena L. Crowell <notifications@github.com>
Отправлено: 8 сентября 2020 г. 16:29
Кому: HelenaLC/CATALYST
Копия: Svetlana Kalmykova; Author
Тема: Re: [HelenaLC/CATALYST] prepData() function is abscent (#136)


Closed #136<https://github.com/HelenaLC/CATALYST/issues/136>.

-
